PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,7 @@
+## Description
+
+## Test Plan
+
+## Checklist
+- [ ] Detox tests pass (only Heap employees are able run these)
```

Comment:
I'm not as familiar with how those checklists work... If someone from outside the org submits a PR, can someone on our team then check those boxes?

Reply:
These are checked by placing an `X` in the box, e.g. 
```
- [X] XYZ thing
```
yields
- [X] XYZ thing

I think it would be reasonable for a Heap employee to edit the PR description to check the detox test box for PRs from outside the org.